pkg/scripts/os.go

```diff
@@ -55,12 +55,11 @@ EOF
 
 sudo mkdir -p /etc/apt/apt.conf.d
 cat <<EOF | sudo tee /etc/apt/apt.conf.d/proxy.conf
-{{- if .HTTPS_PROXY -}}
-Acquire::https::Proxy = "{{ .HTTPS_PROXY }}";
-{{ end }}
-
+{{ if .HTTPS_PROXY -}}
+Acquire::https::Proxy "{{ .HTTPS_PROXY }}";
+{{ end -}}
 {{- if .HTTP_PROXY -}}
-Acquire::http::Proxy = "{{ .HTTP_PROXY }}";
+Acquire::http::Proxy "{{ .HTTP_PROXY }}";
 {{ end }}
```

Comment:
```
{{- if .HTTPS_PROXY }}
Acquire::https::Proxy "{{ .HTTPS_PROXY }}";
{{- end }}
{{- if .HTTP_PROXY }}
Acquire::http::Proxy "{{ .HTTP_PROXY }}";
{{- end }}
```

here you can play with it: https://play.golang.org/p/v_uScHqYcKH

Reply:
thx - your solution looks and behaves cleaner - i will take that